### ISSUE TITLE
Add more features to manage student profile pictures

### ISSUE THREAD

**Issue by thdebay** [academico-sis/academico#123]
Currently, students may upload a picture to their profile.
It would be great to allow them to take picture with webcam during the account creation process.

Also, teachers should be able to see a photo roaster for their classes. Currently, the only place the student pictures are displayed is in the student "show" view.

**Comment by AleksandarMitevski** [comment]
Hello, i'd like to try to solve this issue.

**Comment by thdebay** [comment]
Hello @AleksandarMitevski  and thank you for offering to solve this issue. I have no experience with webcam capture but it should be a fun thing to work on!

I'll assign this issue to you and please let me know if you need any help.

**Comment by AleksandarMitevski** [comment]
If i get stuck somewhere i'll post my confusion here, thanks.

**Comment by AleksandarMitevski** [comment]
Hello @thdebay ,

After a couple of days of trial and error I'm nearly done with the first part of the Issue, I just want to refine the styling and finish up some other things I have in my checklist for the first part. 

One thing I wanted to ask you about was if the user should have the option to choose a profile picture from their files because I added the ability without asking first lol.

Also I have a question relating to HTTP Headers. I'm using the MediaDevices interface and the getUserMedia() method to access the users attached media devices. The documentation says that for the getUserMedia() method to work I need to add the **' Feature-Policy: camera 'self' '** header. I did this by adding **->header('Feature-Policy', "camera 'self' ");** to the method return of **showRegistrationForm()** and changing **return view** to **return response()->view** in the same method, but i want to ask if this is the correct way to go about adding the header. 

Best Regards,
Aleksandar Mitevski

**Comment by thdebay** [comment]
Hello @AleksandarMitevski  and sorry for the delay in replying to you.

It's indeed a very nice idea to let the user upload a picture instead of using the webcam to take it. Can I see your code to check the headers declaration? Do you see the correct headers if you inspect the request which is sent by your browser?

Best regards,
Thomas

**Comment by AleksandarMitevski** [comment]
Hello @thdebay ,

There's no need to be sorry, i understand that other issues take precedence and i'm grateful to be able to contribute.

I added the response header like this per Laravels documentation. I also changed the header from Feature-Policy to Permissions-Policy because it got renamed to that.

![Capture](https://user-images.githubusercontent.com/59975218/91669513-b9aa2e80-eb15-11ea-86ab-21cbabb1f044.PNG)

I checked the response headers i got from loading the registration page and Permissions-Policy showed up among them. 

![Response-Headers](https://user-images.githubusercontent.com/59975218/91669478-92536180-eb15-11ea-93d5-ace6849c54fe.PNG)

Everything seems to be ok to me.
